ecl/eclcc/eclcc.cpp

```diff
@@ -1111,7 +1111,7 @@ void EclCC::processSingleQuery(EclCompileInstance & instance,
             options.includePrivateDefinitions = instance.wu->getDebugValueBool("metaIncludePrivate", true);
             options.onlyGatherRoot = instance.wu->getDebugValueBool("metaIncludeMainOnly", false);
             options.includeImports = instance.wu->getDebugValueBool("metaIncludeImports", true);
-            options.includeExternalUses = instance.wu->getDebugValueBool("metaIncludeExternalUse", true);
```

Comment:
This is a typo I spotted while investigating. None of thee options (other than metaIncludeMainOnly) appear to have been implemented. I'm not sure what the plan was for them?